pkg/api/v20240812preview/openshiftcluster.go

```diff
@@ -290,9 +290,13 @@ type IngressProfile struct {
 
 // PlatformWorkloadIdentityProfile encapsulates all information that is specific to workload identity clusters.
 type PlatformWorkloadIdentityProfile struct {
+	UpgradeableTo              *UpgradeableTo             `json:"upgradeableTo,omitempty"`
```

Comment:
Same as the conversation here:- https://redhat-external.slack.com/archives/C06FRQKUMFF/p1716217461235529
Do we need to add a struct tag `mutable` for this field too?

It has to be only for the external APIs, I suppose.